packages/calling-component-bindings/src/callControlSelectors.ts

```diff
@@ -221,20 +221,16 @@ export const devicesButtonSelector: DevicesButtonSelector = reselect.createSelec
   [getDeviceManager],
   (deviceManager) => {
     return {
-      microphones: removeBlankNameDevices(deviceManager.microphones),
```

Comment:
are the blank name devices coming only from webview, do we need to filter any other devices that might cause issue?